rest/route/reliability_test.go

```diff
@@ -0,0 +1,86 @@
+package route
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/stretchr/testify/suite"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type TaskReliabilitySuite struct {
+	suite.Suite
+}
+
+func configureTaskReliability(disabled bool) error {
+	flags := &evergreen.ServiceFlags{}
```

Comment:
If this is the wrong way to do this, let me know. It seems quite heavy weight to have to actually write to the database but on the plus side it does validate the code path.

Reply:
You're writing to/reading from the db anyway (which isn't heavyweight at all) so I'd just use the Set method on the service flags